<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Page1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Page2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableWidget" name="tableWidget_2"/>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionDelete"/>
    <addaction name="actionLayout_Tab"/>
    <addaction name="actionData_Tab"/>
    <addaction name="actionShow_Dimensions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionTips"/>
   </widget>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionline"/>
   <addaction name="actionRect"/>
   <addaction name="actionEline"/>
   <addaction name="actionErect"/>
   <addaction name="actionSelect"/>
   <addaction name="actionsetScale"/>
  </widget>
  <action name="actionRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>rect</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Draw a rectangular plot&lt;/p&gt;&lt;p&gt;using a corner rectangle&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>line</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Draw a line element, multiple&lt;/p&gt;&lt;p&gt;connected lines form a plot&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clear drawing mode &lt;/p&gt;&lt;p&gt;and select images&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="Del">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Eline</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Exact Line: Set exact length and angle from the clicked location&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionErect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Erect</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exact Rectangle: Set exact dimesnions from the top left corner&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionsetScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>setScale</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select a line and &lt;/p&gt;&lt;p&gt;declare its length&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionTips">
   <property name="text">
    <string>Tips</string>
   </property>
  </action>
  <action name="actionData_Tab">
   <property name="text">
    <string>Data Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLayout_Tab">
   <property name="text">
    <string>Layout Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionShow_Dimensions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Dimensions</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
